lib/test-endpoint.sh

```diff
@@ -2,13 +2,26 @@
 
 unset DD_LOGS_VALID_ENDPOINT
 DATADOG_DIR="${DATADOG_DIR:-/home/vcap/app/datadog}"
+DD_US_API_SITE="https://api.datadoghq.eu/api/"
+DD_EU_API_SITE="https://api.datadoghq.us/api/"
+DD_API_SITE=${DD_US_API_SITE}
+DD_USE_EU=false
```

Comment:
Could we do without this? It seems redundant a little.

Reply:
It lets me not explicitly string match in more than one place, which I think is a bit cleaner, with the expense of this variable.